Src/MergeDoc.cpp

```diff
@@ -2028,11 +2026,46 @@ m_ptBuf[file]->SetLineFlag(i, LF_INVISIBLE, false, false, false);
 
 	m_diffList.ConstructSignificantChain();
 
-	// Used to strip trivial diffs out of the diff chain
-	// if m_nTrivialDiffs
-	// via copying them all to a new chain, then copying only non-trivials back
-	// but now we keep all diffs, including trivial diffs
+	// Buffers `m_ptBuf[]` may have a final line entry that is <null>.
+	// (a) If ALL buffers have that final <null>, remove them all.
+	// (b) If only SOME have that final <null>, change those <null>
+	//		lines into proper ghost-image lines.
+	// Note too: By this point all buffers must have the same number 
+	//		of line entries; eventual buffer processing typically
+	//		uses the line count from `m_ptBuf[0]` for all buffer processing.
 
+	int nFinalNullLines = 0;	
```

Comment:
This code block seems to be the cause of # 106 problem.
Also, with this code block, EOL on the last line can not be deleted by DEL key etc.
I did not find a good fix for this, so I would like to disable this code.